CHANGELOG.rst

```diff
@@ -2,10 +2,11 @@
 Changelog for package greenzie_ntrip_client
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1.2.2 (2023-02-14)
+1.2.2 (2023-03-14)
```

Comment:
Was this intentional?

Reply:
Yeah, I am just copying what Reed did when he did his PR into this repo.